nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -72,7 +72,7 @@ def _ml_parametrization(
                 keras.layers.Dense(64, activation=activation, input_shape=(self.X_train.shape[1],)),
                 keras.layers.Dense(1)
             ])
-            regr.compile(optimizer=solver, loss='mse', metrics=['mae'])
+            regr.compile(optimizer=solver, loss='mse', metrics=['mae'], verbose=0, epochs=350)
```

Comment:
No you can't do that here.
You have to do it in the fit function (lines 82 and 90) without messing up the sklearn fit. 

l.82:
```python
if regressor == "kerasDenseNN":
    regr.fit(self.X_train, self.y_train, verbose=0, epochs=350)
else:
    regr.fit(self.X_train, self.y_train)
```

l.90:
```python
if regressor == "kerasDenseNN":
    regr.fit(X, y, verbose=0, epochs=350)
else:
    regr.fit(X, y)
```